src/RedisPrefixTrait.php

```diff
@@ -94,4 +94,44 @@ protected function getPrefix() {
     return $this->prefix;
   }
 
+  /**
+   * From the given timestamp, with arbitrary increment as decimal, get
+   * the decimal value
+   *
+   * @param int|string $timestamp
+   *   "TIMESTAMP[.INCREMENT]" string
+   *
+   * @return string
+   *   "TIMESTAMP.INCREMENT" string.
+   */
+  public function getNextIncrement($timestamp = null) {
+
+    if (!$timestamp) {
+      return time() . '.000';
+    }
+
+    if (false !== ($pos = strpos($timestamp, '.'))) {
```

Comment:
Please note that I totally violated the coding standards in the 7.x-3.x version to fit with PSR-0 I mostly use, but because I will probably give you maintainer access for the 8.x branch I tried to kept Drupal standards for 8.x, I will fix it.